packages/greencheck/src/Sitecheck/Cache.php

```diff
@@ -106,15 +127,18 @@ public function getCacheType()
      */
     public function getCacheDriver($cachetype)
     {
+
         if ('memcache' == $cachetype) {
             $memcache = new \Memcache();
-            $memcache->connect('127.0.0.1', 11211);
+            $memcachehost = $this->getMemcacheHost();
+            $memcache->connect($memcachehost, 11211);
```

Comment:
you can make these a 1 liner : $memcache->connect($this->getMemcacheHost(), 11211)